src/scripts.js

```diff
@@ -90,18 +90,21 @@ const setTripsDestination = () => {
   let destinationObj = destinationRepo.findDestination(trip.destination);
   trip.destination = destinationObj;
   });
-}
+};
 
 const findYearlySpent = (travelersTrips) => {
   let yearlySpent = 0;
   travelersTrips.forEach(trip => {
     if(trip.date.includes('2022')) {
-      yearlySpent += parseInt(trip.destination.lodgingCost + trip.destination.flightCost)
+      let totalLodging = parseInt((trip.destination.lodgingCost * trip.travelers) * trip.duration)
+      let totalFlight =  parseInt(trip.destination.flightCost * trip.travelers)
+      let agentFee = (totalLodging + totalFlight) * .10
+      yearlySpent += totalLodging + totalFlight + agentFee
       displayYearlySpent(yearlySpent)
```

Comment:
You have this function call inside AND outside of your iterator. Is that intentional?

Reply:
this is fixed with the reduce.. now there isn't a need to call it twice